source/plugins/go/src/oms.go

```diff
@@ -465,6 +469,23 @@ func updateContainerImageNameMaps() {
 	}
 }
 
+func fetchContainerLogV2FromDCR() {
+	for ; true; <-ContainerLogV2DCRTicker.C {
+		//wait for 2 minutes for initialize
+		time.Sleep(2 * time.Minute)
```

Comment:
This is running on different thread,  adding sleep here doesn't help main thread where the postDataHelper running. you still see the initial log records being ingested to containerlog table and subsequent records will be ingested to containerlogv2 table.